containers/ecr-viewer/docker-compose.yaml

```diff
@@ -7,28 +7,37 @@ services:
       dockerfile: Dockerfile
       args:
         - APP_VERSION=1.0
-    ports:
-      - "3000:3000"
+    network_mode: host
```

Comment:
how's the performance?

Reply:
It's totally fine! From my testing it was the OTEL communication that was causing the timeout. I'm not sure why the docker network caused that timeout.